Week 1 Quiz Answers

```diff
@@ -0,0 +1,112 @@
+The <p> tag is both semantic and presentational, because browsers and developers not only understand what information will be contained within the <p> tag, but a browser generally understands how to present the tag (even without more specific styling).
+The <i> tag is not semantic, for it is a style. Simply seeing the <i> tag will not tell a developer or browser any more information other than that information will be italicized.
+Furthermore, it is not semantic because even italicized words could have different meanings (i.e. a book title vs. emphasis).
+
+Question #2
+
+What is the purpose of the alt attribute? e.g:
+
+<img src='https://jesse.sh/img/me.jpg' alt='Jesse Shawl'/>
+
+//Alt tags are useful for both developers and users. For developers, the alt attribute aids him/her as he/she reads/visualizes code, for it gives an image a title and will display that title even if the image fails to load in the browser.
+It is especially useful for users with disabilities such as blindness. Certain text readers and other tools will say the alt attribute out loud, permitting a user with a disability to understand that an image is located on the page.
+
+Question #3
+
+What is the purpose of the <head></head> tag in HTML?
+
+In general, everything that will not be seen by the user is placed in the <head> tag.
+To be more specific, the <head> tag collects meta data and other information about the document.
+It is where a developer will place definitions of links and stylesheets (CSS, JS) essential to the functionality and display of the document.
+It is also essential to SEO, for a developer could place keywords and other useful SEO tags/content/definitions within the <head> element.
+
+CSS
+
+Question #4
+
+In the spaces below, write the CSS property that best matches the given description.
+
+The choices are border, outline, padding, and margin. You will leave two spaces blank.
+
+Your Answer:
+
+MARGIN: Defines the distance between an element's border and adjacent elements' borders.
+BORDER: Inserts a "wall" around an element.
+PADDING: Defines the distance between an element's content and its border.
+___: Defines the width of an element.
+OUTLINE: Overlays a "wall" on top of an element.
+___: Defines the distance between the center of an element and the center of the adjacent element.
+
+Question #5
+
+What does the following selector do? ul.dropdown > li?
+
+Select 1:
+
+[X] Selects all li's which are directly inside a ul of class dropdown (children)
+[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[] Selects all ul's of class dropdown, only if their children are exclusively li's
+Question #6
+
+Identify the three places CSS can go, and rank them in terms of specificity:
+
+1. linked stylesheet
+2. Embed CSS on tags themselves
+3. Create a CSS block in web page (typically between <head> tags)
+
```

Comment:
Not Quite...

1 Inline but within the HTML element itself `style=`
2 Within` <style></style>` tags in the `<head>` of an html document
3 In an external or "separate css" file or stylesheet